scripts/run-nix-tests.sh

```diff
@@ -21,4 +21,6 @@ cd ../tests/nix_tests/hardhat
 npm install
 npm run typechain
 cd ..
-pytest -vv -s
+
+# shellcheck disable=SC2086
```

Comment:
I'd prefer to have an explainer of what is disabled with these shellcheck escapes, e.g.
```suggestion
# shellcheck disable=SC2086 -- we want to pass the arguments as separate words instead of one string
```